docs/Homebrew-on-Linux-and-Windows-10.md

```diff
@@ -1,8 +1,11 @@
-![Linuxbrew logo](https://linuxbrew.sh/images/linuxbrew-256x256.png)
+---
+title: Homebrew on Linux (and Windows 10)
+image: https://brew.sh/assets/img/linuxbrew.png
+redirect_from: /Linuxbrew
+---
+<img src="https://brew.sh/assets/img/linuxbrew.png" alt="Linuxbrew logo" style="min-width:128px" height="128" width="128">
 
-# Linuxbrew
-
-The Homebrew package manager may be used on Linux and Windows 10, using [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about). Homebrew is referred to as Linuxbrew when running on Linux or WSL. It can be installed in your home directory, in which case it does not use *sudo*. Linuxbrew does not use any libraries provided by your host system, except *glibc* and *gcc* if they are new enough. Linuxbrew can install its own current versions of *glibc* and *gcc* for older distributions of Linux.
+The Homebrew package manager may be used on Linux and Windows 10, using [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about). Homebrew was formerly referred to as Homebrew when running on Linux or WSL. It can be installed in your home directory, in which case it does not use *sudo*. Homebrew does not use any libraries provided by your host system, except *glibc* and *gcc* if they are new enough. Homebrew can install its own current versions of *glibc* and *gcc* for older distributions of Linux.
```

Comment:
"Homebrew was formerly referred to as Homebrew"? Should the 2nd instance be Linuxbrew?

Reply:
Yes, thanks.